data/src/main/java/com/github/abdallahabdelfattah13/data/DataInjection.kt

```diff
@@ -11,6 +17,26 @@ import com.github.abdallahabdelfattah13.domain.repositories.feed.FeedRepository
  */
 object DataInjection {
 
+    private lateinit var mFeedDatabase: FeedDatabase
+
+    @JvmStatic
+    fun provideFeedInMemory(): InMemoryFeedRepo = InMemoryFeedRepoImpl()
+
+    @JvmStatic
+    fun provideDatabase(context: Context): FeedDatabase {
+        mFeedDatabase = Room.databaseBuilder(
+            context,
+            FeedDatabase::class.java,
+            "feed.db"
+        )
+            .build()
```

Comment:
This indentation is a little bit weird. I think you can move this call to the same lane you close the parenthesis.

Reply:
Ok. Done